### PR TITLE
[BFN] Refactoring and adding some functions of Thermal class (set and get thresholds and etc.)

### PR DESCRIPTION
**Pull request by VadymYashchenko**
#### Why I did it
I revise functions for correct working

#### How I did it
I added default_shresholds.json file  and refactor and added functions for correct work with thresholds

#### How to verify it
I verified it with thermal tests on Newport device

#### Description for the changelog

Refactoring and adding some functions of Thermal class (set and get thresholds and etc.)

### CODE REVIEW

**Comment by VadymYashchenko** [comment]
@lguohan , @sujinmkang , @prgeor and  @vdahiya12 could you make review of this merge request?

**Review by akokhan — DISMISSED** [review]

**Comment by VadymYashchenko** [comment]
@lguohan , @sujinmkang , @prgeor and @vdahiya12 could you make review of this merge request?

**Comment by akokhan** [comment]
@prgeor , could you please take a look? Thanks

**Comment by VadymYashchenko** [comment]
@lguohan , @sujinmkang , @prgeor and @vdahiya12 could you make review of this merge request?

**Comment by VadymYashchenko** [comment]
@lguohan , @sujinmkang , @prgeor and @vdahiya12 could you make review of this merge request?

File: device/barefoot/x86_64-accton_as9516_32d-r0/thermal_thresholds.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        {
+            "psu_driver-i2c-7-59:psu2-temp1" : [60.0, 50.0, 11.0, 1.0]
+        },
+        {
+            "psu_driver-i2c-7-59:psu2-temp2" : [60.0, 50.0, 11.0, 1.0]
+        },
+        {
+            "psu_driver-i2c-7-59:psu2-temp3" : [60.0, 50.0, 11.0, 1.0]
+        },
+		{
```
Comment (akokhan) [inline device/barefoot/x86_64-accton_as9516_32d-r0/thermal_thresholds.json:51]: Please fix indentation.

Comment (VadymYashchenko) [reply device/barefoot/x86_64-accton_as9516_32d-r0/thermal_thresholds.json]: done

**Review by akokhan — APPROVED** [review]

**Comment by VadymYashchenko** [comment]
@lguohan , @sujinmkang , @prgeor and @vdahiya12 could you make review of this merge request?

**Comment by akokhan** [comment]
@prgeor , please approve and merge

**Pull request merged**